templates/service/.env.example

```
@@ -0,0 +1,42 @@
+DEBUG_HEADERS=false
+MCP_CPU_LIMIT=0.50
+MCP_MEMORY_LIMIT=256M
+MCP_MEMORY_RESERVATION=128M
+MCP_NETWORK_EXTERNAL=traefik
+MCP_BUILD_CONTEXT=..
+MCP_DOCKERFILE=services/__SERVICE_NAME__/Dockerfile
+
+# Derived manifest overrides (optional)
+MCP_SERVER_NAME=__SERVICE_NAME__
+MCP_SERVER_VERSION=0.1.0
+MCP_NAME_HUMAN=${__SERVICE_ENV_PREFIX___SERVICE_NAME_HUMAN}
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The MCP_NAME_HUMAN key should go before the MCP_SERVER_NAME key

(UnorderedKey)